<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewQtGuiClass</class>
 <widget class="QWidget" name="NewQtGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>导入数据</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>圆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>线段</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>射线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>直线</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择数据类型</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
